primefaces/src/main/java/org/primefaces/util/FileUploadUtils.java

```diff
@@ -237,16 +237,10 @@ protected static String convertJavaScriptRegex(String jsRegex) {
         return LangUtils.substring(jsRegex, start, end);
     }
 
-    private static boolean isValidFileContent(PrimeApplicationContext primeAppContext, FileUpload fileUpload, String fileName,
+    private static boolean isValidFileContent(PrimeApplicationContext primeAppContext, String accept, String fileName,
```

Comment:
acceptTypes

Reply:
Not sure about this one
Its just called accept in HTML and on p:fileUpload/h:inputFile, we should propably stick to it.

Reply:
It's a lot more explicit than just accept, it's also important to show plurarity

Reply:
how is it more then just accept?
Its a list of allowed content-types, same like the HTML accept attribute.

Reply:
also for me its very important to stick with native HTML names; just think about div vs h:outputPanel

Reply:
Exactly, it's a list... based on what you modify after, how do you make a difference between acceptPart and accept? That's not really clear at that point... That's why I would emphase plurality with acceptTypes, and then use acceptType to emphase it is one accept-chunk from accepTypes string